src/main/java/com/kosherjava/zmanim/hebrewcalendar/JewishCalendar.java

```diff
@@ -661,7 +661,7 @@ public int getYomTovIndex() {
 			if (day == 6 || (day == 7 && !inIsrael)) {
 				return SHAVUOS;
 			}
-			if ((day == 7 && inIsrael) || (day == 8 && !inIsrael)) {
+			if (day == 7 || day == 8 && !inIsrael) {
```

Comment:
same here, we don't need to worry about checking inIsrael because we checked it above already